internal/broker/testdata/TestConcurrentIsAuthenticated/golden/first_auth_starts_and_finishes_before_second/first_auth

```diff
@@ -1,3 +1,3 @@
-access: granted
-data: '{"userinfo":{"name":"user-timeout-0","uuid":"saved-user-id","dir":"/home/user-timeout-0","shell":"/usr/bin/bash","gecos":"user-timeout-0","groups":[{"name":"remote-group","ugid":"12345"},{"name":"linux-local-group","ugid":""}]}}'
+access: denied
+data: '{"message":"authentication failure: password file does not exist"}'
```

Comment:
Be more careful when updating golden files. This is not the expected behavior of the test...